text/0039-service-overhaul.md

```diff
@@ -0,0 +1,196 @@
+***This RFC is dependent on [RFC 38](0038-protocol-api.md).***
+
+# Summary
+
+- Make OBS able to accept third-party service plugins
+- Attribute a id for each services rather than a common one
```

Comment:
How will id collisions be prevented?

Reply:
Id collision are prevented when registering and this is already in OBS Studio
https://github.com/obsproject/obs-studio/blob/master/libobs/obs-module.c#L790